### DISCUSSION TITLE
checkbox in table row

### DISCUSSION THREAD

**Discussion by ionv06** [sqlpage/SQLPage#736] [General]
Hi! I can't handle the handlebars syntax, I would like to add `<td><input type="checkbox" /></td>` to the beginning of each row in a table.handlebars template.
I would like to be able to call the table with a top-level parameter, 'select table as component, true as checkbox'. Can someone help me?

**Comment by lovasoa** [comment]
Hello ! The default table component has a lot of features. You may want to create your own simpler table component if you want to add features to it

**Reply by ionv06** [reply]
I know that the table component has many features, I wanted them all plus one more - a checkbox on each row.

**Reply by lovasoa** [reply]
Yes, then you'll have to deal with the complexity of the existing component. If you want to add a column to each line, you can do it here:

https://github.com/sqlpage/SQLPage/blob/63f1ff5e6f64b48c9ea424c70b28afa62d72f7b9/sqlpage/templates/table.handlebars#L29-L30

https://github.com/sqlpage/SQLPage/blob/63f1ff5e6f64b48c9ea424c70b28afa62d72f7b9/sqlpage/templates/table.handlebars#L49-L50
